frontend/src/component/HeaderNav.tsx

```diff
@@ -1,12 +1,17 @@
 import {NavLink} from "react-router-dom";
 import icon from "../sawIcon.png";
+import Secret from "../site/Secret";
+import React from "react";
 
-export default function HeaderNav() {
-
+export default function HeaderNav(props: {
+                                      authenticationChanged: () => void,
+                                      username: string | undefined,
+                                  }
+) {
     return <header>
         <div id="logo"><img src={icon} alt={"icon"}/></div>
         <NavLink className={"navLink"} to={"/"}><h1> Product - Gallery </h1></NavLink>
```

Comment:
There is only one h1 allowed per page, having it link to another page is kinda weird, and if it is the same page, why link to it? 🤔